docs/src/test/java/docs/http/javadsl/server/BlockingInHttpExamples.java

```diff
@@ -60,7 +61,7 @@ void blockingHttpDedicatedDispatcher() {
         //#blocking-example-in-dedicated-dispatcher
         // GOOD (the blocking is now isolated onto a dedicated dispatcher):
         final Route routes = post(() -> {
-            final MessageDispatcher dispatcher = system.dispatchers().lookup("my-blocking-dispatcher");
```

Comment:
the existing code compiles for me - why change it?

Reply:
I originally thought changes were necessary because I couldn't paste this code into a sample project. On closer inspection (and realizing this file does compile 🤦) I found my original mistake. In my sample project I was using the `typed` actor system which does have API changes. This is also the source of the odd `m` prefix in dispatcher names: the typed ActorSystem apply method demands a name. I will revert these changes and regenerate screenshots using the classic actor system.